SU-HSDO/suhumsci
# SHS-5574: Implementation: Refactor some grid CSS/mixins

# READY FOR REVIEW

## Summary
- Remove flexbox fallback for grids
- Remove unnecessary mixins
- Update Percy CLI version and change default test to consolidate Colorful and Traditional into one single test

## Need Review By (Date)
05/03

## Urgency
medium

## Steps to Test
1. Check Percy VRT to confirm that there are no visual changes on the site: https://percy.io/da667d23/suhumsci/builds/34039580

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
